src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunctionSet.cs

```diff
@@ -28,45 +32,68 @@ internal class TexlFunctionSet
         // Count of functions
         internal int _count;
```

Comment:
with new Count() method - should this field now be private? 
Ie, should all callers go through Count() now?

Reply:
yes we can make it private